lib/mintacoin/accounts/accounts.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Mintacoin.Accounts do
+  @moduledoc """
+  This module is the responsible for the CRUD operations for accounts and also for the aggreate operations within the accounts context.
+  """
+
+  alias Ecto.Changeset
+  alias Mintacoin.{Account, Accounts.Cipher, Accounts.Keypair, Repo}
+
+  @type address :: String.t()
+  @type seed_words :: String.t()
+  @type signature :: String.t() | nil
+  @type parameter :: keyword()
+  @type error :: Changeset.t() | :not_found | :decoding_error | :bad_argument
+
+  @spec create :: {:ok, Account.t()} | {:error, error()}
+  def create do
+    Keypair.build_signature_fields()
+    |> (&Account.create_changeset(%Account{}, &1)).()
+    |> Repo.insert()
+  end
+
+  @spec retrieve(address :: address()) :: {:ok, Account.t() | nil} | {:error, error()}
+  def retrieve(address) when is_binary(address) do
+    {:ok, Repo.get_by(Account, address: address)}
+  end
```

Comment:
@jhonatan-kmt @FelipeGuzmanSierra IMO, we can use here the single line notation:

```suggestion
  def retrieve(address) when is_binary(address), do: {:ok, Repo.get_by(Account, address: address)}
```